packages/contracts/src/dollar/core/UbiquityDollarManager.sol

```diff
@@ -81,7 +75,7 @@ contract UbiquityDollarManager is AccessControl {
         // to be removed
 
         TWAPOracleDollar3pool oracle = TWAPOracleDollar3pool(twapOracleAddress);
-        oracle.update();
```

Comment:
@zgorizzo69 any thoughts on this

Reply:
@hashedMae I think we can keep it as a good practice just to make sure that the newly added oracle is updated 
@pavlovcik not that TWAP oracle is a facet in the diamond final implementation so updating its state will be done inside the diamond init